apps/test-netzer-encoder/src/NetzerEncoder.cpp

```diff
@@ -0,0 +1,29 @@
+#include "NetzerEncoder.h"
+#include "mbed.h"
+
+NetzerEncoder::NetzerEncoder(int frequency) : freq(frequency){}
+
+void NetzerEncoder::spi_callback_debug(const int events){
+	printf("%d", events);
+}
+
+void NetzerEncoder::spi_read(){
+	
+	// Callback function to be executed during reads
+	event_callback_t spi_read_callback = NetzerEncoder::spi_callback_debug;		
+
+	spi.format(16,3);				// May have to be 8 bits?
+	spi.frequency(this->freq);
+
+	ThisThread::sleep_for(3ms);		// Delay
+
+	// Read from spi constantly
+	while (true){
+		spi.transfer(tx_buffer, WORDS, rx_buffer, WORDS, spi_read_callback, NULL);
+	}
+
+}
```

Comment:
read function should read once. no whilte(true) inside a function. you'll see what the function signature should look like in the Encoder.h i have in my PR but it returns a bool and takes in a float by reference

Reply:
my latest implementation is similar to Cindy's AEAT-6012 encoder, but  this is is different to Encoder.h right?